src/code/z_camera.c

```diff
@@ -7170,24 +7176,25 @@ s32 Camera_CheckWater(Camera* camera) {
             if (*quakeId != 0) {
                 Quake_RemoveFromIdx(*quakeId);
             }
-            camera->unk_150 = 0;
-            camera->unk_152 = 0;
+            camera->waterDistortionTimer = 0;
+            camera->distortionFlags = 0;
         }
         Audio_SetExtraFilter(0);
     }
     //! @bug: doesn't always return a value, but sometimes does.
 }
 
 /**
- * Sets the room to be hot camera quake flag
+ * Sets the distortion to type 1 for a hot-room
+ * Gives the hot-room a small mirage-like appearance
```

Comment:
may just be me but I would move this comment near where `DISTORTION_HOT_ROOM` is defined, or where the distortion is actually computed (in `Camera_UpdateDistortion`)